core/src/main/resources/hudson/slaves/JNLPLauncher/main_es.properties

```diff
@@ -20,7 +20,6 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-slaveAgentPort.disabled=El puerto TCP para los agentes via JNLP está deshabilitado.
```

Comment:
Renaming key and deleting old translations since there is now a significant extra clause about WebSockets.